skype/skype-ps/skype/Get-CsOnlineVoiceUser.md

```diff
@@ -193,6 +202,11 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+```yaml
+The below updates are applicable to TeamsOnly tenants using TPM modules 3.0 onwards. 
+LiscenceState has been deprecated because it is no longer relevant to TeamsOnly tenants
+```
+
```

Comment:
Same comments apply here. 

What does **LicenseState** have to do with the **Identity** parameter? This seems out of place.